src/kafka/encode_decode_test.go

```diff
@@ -45,7 +45,7 @@ func TestDecoder_JsonMessageToRecord_MalformedJson(t *testing.T) {
 		Partition: 1,
 		Offset:    54,
 		Timestamp: time.Now(),
-	})
+	}, false)
```

Comment:
can you add some tests with the `withKeyAndValue` param set to true?

Reply:
sure